src/utils/QCCalculator.cpp

```diff
@@ -144,11 +147,23 @@ class TOPPQCCalculator :
     String inputfile_consensus = getStringOption_("consensus");
     String inputfile_raw = getStringOption_("in");
     String outputfile_name = getStringOption_("out");
-    String contactName = getStringOption_("name");
-    String contactAddress = getStringOption_("address");
+    String contact_name = getStringOption_("name");
+    String contact_address = getStringOption_("address");
     String description = getStringOption_("description");
     String label = getStringOption_("label");
     bool remove_duplicate_features(getFlag_("remove_duplicate_features"));
+    
+    // ensure output file hase valid extension
+    FileTypes::Type out_type = FileTypes::nameToType(getStringOption_("out_type"));
+    if (out_type == FileTypes::UNKNOWN)
+    {
+      FileHandler fh;
+      out_type = fh.getTypeByFileName(outputfile_name);
+    }
+    if (out_type != FileTypes::QCML and out_type != FileTypes::MZQC) 
```

Comment:
this should not be needed, since the restriction is already enforced by TOPPBase
https://github.com/OpenMS/OpenMS/pull/5282/files#diff-4d40a77e3e4025cbf9759cb22f4559b8ca1441f0f02c6bd347f09dad9e342eb3R128